### DISCUSSION TITLE
Text look bolder than other terminals

### DISCUSSION THREAD

**Discussion by arcogabbo** [wezterm/wezterm#3000] [Q&A] [answered]
Hi guys,
I'm enjoying this terminal emulator so thank you guys for the development.
The only issue is that the font i use looks bolder in wezterm(version `20221119-145034-49b9839f`) compared to kitty terminal.
The font is the same for the two terminals as the size and to me the way kitty renders it is more pleasant to the eyes.
![image](https://user-images.githubusercontent.com/22002572/214268645-3c005ec3-56ad-4c5c-acc2-76dbeeae22e9.png)
`wezterm is on the left`
I already tried to change the weight and other configuration like font_antialias and cell_width with no success.
Any help is appreciated.

**Comment by wez** [comment]
`font_antialias` was removed long ago. I would suggest looking at https://wezfurlong.org/wezterm/config/lua/config/freetype_load_target.html which replaced it; specifically, turning setting it to `HorizontalLcd` will turn on subpixel AA which can make the text look softer.

**Reply by arcogabbo** [reply]
`HorizontalLcd` doesn't affect the font softness. However `freetype_load_flags = "NO_HINTING"` did the trick. Now the font looks exactly as soft as how kitty was rendering it.

**Comment by arcogabbo** [comment, marked as answer]
`freetype_load_flags = "NO_HINTING"` did the trick. Now the font looks exactly as soft as how kitty was rendering it.

**Reply by rafo** [reply]
So, shouldn't freetype_load_flags = "NO_HINTING" be a default setting?

**Reply by wez** [reply]
The default setting is the default setting recommended by the freetype library authors. You do usually want hinting enabled. https://freetype.org/freetype2/docs/hinting/subpixel-hinting.html has more information about hinting in freetype.

If your taste is different, then you can easily configure this for yourself.

**Reply by rafo** [reply]
Yes, you are right!
